expression/builtin_time_vec.go

```diff
@@ -1989,11 +1989,93 @@ func (b *builtinCurrentDateSig) vecEvalTime(input *chunk.Chunk, result *chunk.Co
 }
 
 func (b *builtinMakeTimeSig) vectorized() bool {
-	return false
+	return true
+}
+
+func (b *builtinMakeTimeSig) getVecIntParam(arg Expression, input *chunk.Chunk, col *chunk.Column) (err error) {
+	if arg.GetType().EvalType() == types.ETReal {
+		err = arg.VecEvalReal(b.ctx, input, col)
+		if err = handleInvalidTimeError(b.ctx, err); err != nil {
```

Comment:
I just found that, the function `handleInvalidTimeError` is used incorrectly in the vectorized evaluation, not only here. It may erase the error returned by the eval args.

Reply:
Any plan to handle this issue? cc @qw4990 @SunRunAway

Reply:
We does not need to handle invalide time error here. It's `arg`'s responsibility in vectorized version.